File: python/0075.py
class Solution:
    def sortColors(self, nums):
        """
        :type nums: List[int]
        :rtype: void Do not return anything, modify nums in-place instead.
        """
        r, g, b = 0, 0, 0
        for i in nums:
            if i == 0:
                r += 1
            elif i == 1:
                g += 1
            elif i == 2:
                b += 1
        j = 0
        for i in range(r):
            nums[j] = 0
            j += 1
        for i in range(g):
            nums[j] = 1
            j += 1
        for i in range(b):
            nums[j] = 2
            j += 1

File: rust/src/bin/0075.rs
struct Solution;

impl Solution {
    pub fn sort_colors(nums: &mut Vec<i32>) {
        let mut cnt = [0; 3];
        for i in nums.iter() {
            cnt[*i as usize] += 1;
        }
        let mut j = 0;
        for i in 0..3 {
            for k in 0..cnt[i] {
                nums[j] = i as i32;
                j += 1;
            }
        }
    }
}

fn main() {}